services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowValidationNegativeSpec.groovy

```diff
@@ -161,4 +166,44 @@ class FlowValidationNegativeSpec extends BaseSpecification {
         def switches = topology.getActiveSwitches()
         return [switches.first(), switches.first()]
     }
+
+    @Unroll
+    def "Unable to #action flow with non-existent flow id"() {
```

Comment:
Sorry, how there can be flow with non-existent flow ID. I would rephrase it to:

Unable to #action a non-existent flow

Reply:
Also, it can be a single test:

```
@Unroll
def "Unable to #action flow with non-existent flow id"() {
    assumeFalse("Ignored due to issue #2027", action == "validate")
    ...
    northbound."${action}Flow"(NON_EXISTENT_FLOW_ID)
    ...

    where:
    action        | message
    "get"         | "Can not get flow: Flow $NON_EXISTENT_FLOW_ID not found"
    "synchronize" | "Could not synchronize flow: Flow $NON_EXISTENT_FLOW_ID not found"
    "reroute"     | "Could not reroute flow: Flow $NON_EXISTENT_FLOW_ID not found"
    "validate"    | "Can not validate flow: Flow $NON_EXISTENT_FLOW_ID not found"
```